pkg/kinflate/util/gvkn_sort.go

```diff
@@ -0,0 +1,28 @@
+/*
```

Comment:
This file is copied from `pkg/kinflate/gvkn_sort.go`
The original copy will be deleted after `kinflate` switched to use `pkg/kinflate/util`.